tool/e2e-tests/helpers.ts

```diff
@@ -71,17 +75,43 @@ export async function sendPrescriptionUserJourney(
 
 export async function sendBulkPrescriptionUserJourney(
   driver: ThenableWebDriver,
-  loadExamples: (driver: ThenableWebDriver) => Promise<void>,
+  fileInfo: FileUploadInfo,
   successfulResultCountExpected: number
 ): Promise<void> {
   await loginViaSimulatedAuthSmartcardUser(driver)
   await setMockSigningConfig(driver)
   await createPrescription(driver)
-  await loadExamples(driver)
+  await loadTestData(driver, fileInfo)
   await sendPrescription(driver)
   await checkBulkApiResult(driver, successfulResultCountExpected)
 }
 
+export async function prescriptionIntoClaimedState(driver: ThenableWebDriver, fileUploadInfo: FileUploadInfo): Promise<void> {
+  const prescriptionId = await sendPrescriptionSingleMessageUserJourney(driver, fileUploadInfo)
+  await releasePrescriptionUserJourney(driver)
+  await dispensePrescriptionUserJourney(driver)
+  await claimPrescriptionUserJourney(driver)
+  await checkMyPrescriptions(driver, "Claimed Prescriptions", prescriptionId)
+}
+
+export async function prescriptionIntoCanceledState(driver: ThenableWebDriver, fileUploadInfo: FileUploadInfo): Promise<void> {
+  await sendPrescriptionSingleMessageUserJourney(driver, fileUploadInfo)
+  await cancelPrescriptionUserJourney(driver)
+}
+
+export async function sendPrescriptionSingleMessageUserJourney(
```

Comment:
Can the File Upload framework be updated so that it can handle a default journey and a bulk journey, as well as the current single file journey? Could help clean up a few similar functions at the top of this file.

Reply:
Yeah I agree we can clean helpers up further in a future ticket